doc/modules/kernel_approximation.rst

```diff
@@ -132,6 +132,56 @@ with the approximate feature map provided by :class:`RBFSampler` to yield an app
 feature map for the exponentiated chi squared kernel.
 See the [VZ2010]_ for details and [VVZ2010]_ for combination with the :class:`RBFSampler`.
 
+.. _intersection_kernel_approx:
+
+Intersection Kernel
+-------------------
+
+The intersection kernel is a kernel on histograms, often used in computer vision.
```

Comment:
Could you please reduce your line wrapping to 80 chars or so. I know this is violated above, but still...